pycm/pycm_util.py

```diff
@@ -781,3 +781,31 @@ def inner_function(*args, **kwargs):
             raise pycmMatrixError(METRICS_OFF_ERROR)
         return func(*args, **kwargs)
     return inner_function
+
+
+def deprecated(func):
+    """
+    Send a warning regarding decorated function's deprication.
+
+    :param func: input function
+    :type func: function
+    :return: inner function
+    """
+    @wraps(func)
+    def inner_function(*args, **kwargs):
+        """
+        Inner function.
+
+        :param args: non-keyword arguments
+        :type args: list
+        :param kwargs: keyword arguments
+        :type kwargs: dict
+        :return: None
```

Comment:
None?

Reply:
`metrics_off_check`'s `inner_function` docstring is also wrong.
It should be something like `modified function`

Reply:
@alirezazolanvari

Reply:
I think it should be `modified function result` since it's somehow a call for that function.